lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -324,6 +385,47 @@ bool AsyncPipelinedClient::add_gy_final_action_flow(
   return true;
 }
 
+//RPC definition to Send Set Session request to UPF
+ void AsyncPipelinedClient::set_upf_session_rpc(
+     const SessionSet& request,
+     std::function<void(Status, UPFSessionContextState)> callback) {
+   auto local_resp = new AsyncLocalResponse<UPFSessionContextState>(
+       std::move(callback), RESPONSE_TIMEOUT);
+   PrintGrpcMessage(
+       static_cast<const google::protobuf::Message&>(request));
+
+//Cerrors are temporary to check Unit Test Cases and there associated logs
+   std::cerr << __LINE__ << " "<< __FUNCTION__ << " GRPC message sending to Upf client details\n";
+   std::cerr << __LINE__ << " "<< __FUNCTION__ << " SeId "<<request.seid() <<"\n";
+   std::cerr << __LINE__ << " "<< __FUNCTION__ << " Vesion no "<<request.sess_ver_no() <<"\n";
+   //std::cerr << __LINE__ << " "<< __FUNCTION__ << "State"<<request.state() <<"\n";
+   std::cerr << __LINE__ << " "<< __FUNCTION__ << " Pdr list count "<<request.set_gr_pdr_size() <<"\n";
+   for (int i=0; i<request.set_gr_pdr_size(); i++) {
+         const magma::SetGroupPDR pdr_=request.set_gr_pdr(i) ;
+                 std::cerr << __LINE__ << " "<< __FUNCTION__ << " Pdr id "<<pdr_.pdr_id() <<"\n";
```

Comment:
wouldn't be useful to declare a `toString` method attached to those objects `pdr` `far` etc...? 
This we will be able to use them in future debugging.